agents/plugins/apache_status

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+# pylint: disable=W0141
```

Comment:
Bitte die Supressions nur für den Scope machen, für den sie auch wirklich erforderlich sind. Und dann lieber mit ihrem Namen, damit man weiß, was da das Problem ist (ich weiß es hier jetzt nicht).

Reply:
Ist ganz raus, habe den code geändert.

Reply:
mh ok, obwohl die lokalen Tests alle ok waren, knallt der test hier.
Ich schätze wieder wegen der user of buildin function